site/content/news/rdf4j-360.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "RDF4J 3.6.0 released"
+date: 2021-02-22T10:00:00+0100
+layout: "single"
+categories: ["news"]
+---
+We are pleased to announce that RDF4J 3.6.0 is now available. This is a minor release with about XXXX 
+improvements and bug fixes.
+
+For full details, have a look at the [release notes](/release-notes/3.6.0). Below, we briefly highlight some
+of the major improvements.
+
+<!--more-->
+
+## Improved SHACL Implementation
+RDF4J likely sports the worlds only SHACL implementation capable of analyzing your transactional changes.
```

Comment:
I don't think this is quite true anymore, to be honest. For example, Stardog has some support for this kind of thing as well (they call it "Guard Mode").  

Suggestion:

"RDF4J sports one of the world's fastest SHACL implementations capable of analyzing your transactional changes."

Reply:
Stardog is why i developed the ShaclSail. At my previous employer we used it with close to a billion triple. Had validation rules for a couple of the graphs, maybe a few million triples in total. Transaction time just for validating was 500ms.

Reply:
I did include one weasel word (likely) just in case someone has gone under my radar.

Reply:
My point is that there are other implementations _capable_ of transactional validatio. We're not the only one that can do it, though (perhaps) the fastest (and even that I wouldn't want to boast about publicly until we'd a proper comparitive analyis).

Reply:
There are no other that analyze the contents of the transaction. Last I checked we were 50x faster than Stardog with 500 000 triples, which does not analyze the contents but instead just reruns the full suite of shacl validation rules.

Reply:
I know Neo4J does the same. And Jena also only supports full validation, and I'm not sure if they have transactional support. Allegrograph seems to only have an CLI for validating SHACL and no transactional support. I'm not sure about TopQuadrant, but I'm sure I would have heard something if they have managed to implement transactional support that analyzes the changes instead of revalidating everything every time.

Reply:
I understand what you're trying to say, I think as a compromise I'll put the word "incrementally" in there somewhere, to make a it little more obvious what our point of differentation is.